Source/WebGPU/WGSL/ConstantRewriter.cpp

```diff
@@ -252,7 +252,10 @@ void ConstantRewriter::materialize(Node& expression, const ConstantValue& value)
     evaluated(expression, value);
 
     const auto& replace = [&]<typename Literal, typename Value>() {
-        auto& node =  m_shaderModule.astBuilder().construct<Literal>(SourceSpan::empty(), std::get<Value>(value));
+        // FIXME: https://bugs.webkit.org/show_bug.cgi?id=262068
+        auto* maybeValue = std::get_if<Value>(&value);
```

Comment:
@tadeuzagallo I filed an bug for this issue because working around the crash like this seems wrong

Reply:
With out diving in to work out where in the AST this is issue it coming from, I'm going to defer to @tadeuzagallo

Reply:
Seems like we fail to compute a constant. It's fine with me to land as is and I can fix it after you land it.